notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>... to describe it completely (x, y, z); or (theta, phi, r) in spherical coordinates.</p><p> </p><p>Love the mentally taxing / computationally intensive part! We’re all machines after all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>